airbyte-webapp/src/components/connectorBuilder/Builder/BuilderSidebar.module.scss

```diff
@@ -0,0 +1,102 @@
+@use "scss/variables";
+@use "scss/colors";
+@use "scss/mixins";
+
+.container {
+  padding: variables.$spacing-sm;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+}
+
+.connectorImg {
+  @include mixins.shadow;
+
+  width: 90px;
+  border-radius: 25px;
+  padding: variables.$spacing-xl;
+  margin-top: 55px;
+}
+
+.connectorName {
+  margin-top: variables.$spacing-md;
+  min-height: 28px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  overflow-x: auto;
+}
+
+.connectorNameText {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.streamsHeader {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 0;
+  margin-top: variables.$spacing-2xl;
+}
+
+.streamsHeading {
+  margin-left: variables.$spacing-md;
+  color: colors.$blue;
+}
+
+.addStreamButton {
+  margin-left: auto;
+  margin-right: variables.$spacing-sm;
+}
+
+.downloadButton {
+  margin-top: auto;
+  width: 90%;
+}
+
+.resetButton {
+  margin-top: variables.$spacing-sm;
+  margin-bottom: variables.$spacing-md;
+  width: 90%;
```

Comment:
I'm not so sure about these 90% widths - is there a reason the buttons are not as wide as the sidebar allows, just like the stream list items itself? It looks cleaner to me:
<img width="954" alt="Screenshot 2022-12-09 at 15 12 38" src="https://user-images.githubusercontent.com/1508364/206721594-b795c94c-167f-4b55-baaa-1d017d2130b8.png">

Reply:
I agree, I think using the full width is better. Fixed